doc/user/separate.xml

```diff
@@ -55,12 +55,12 @@
   and header files differ.  If you build "Controller A" first,
   then "Controller B", on the second build everything would
   have to be rebuilt, because &SCons; sees that the build
-  instructions differ, and thus the targets that depend on those
-  different instructions are not valid for the current build.
+  instructions are different to the ones used to build the
+  existing targets,  and thus the existing targets are invalid.
   Now when you go back and build for "Controller A",
   things have to be rebuilt from scratch again for the same reason.
-  However, if you can separate the places the output files
-  go, this problem can be avoided.
+  However, if you can separate the places the output files go,
```

Comment:
".. the locations of the output files, "?

Reply:
updated

Reply:
Hmmm, come to think of it, I didn't really even intend this change to the user guide to be part of this PR, it was part of a separate fiddle to variantdir things. Must have committed it without thinking. I can split it out if you'd prefer, or we can proceed here.

Reply:
here should be fine.